docs/network-flow-visibility.md

```diff
@@ -199,22 +199,37 @@ exploration.
 
 ### Deployment Steps
 
-To create all the necessary resources in the `elk-flow-collector` Namespace
-and get everything up-and-running, run:
+First download the necessary resources from the Antrea repository. If the deployed
+version of Antrea has a release `<TAG>` (e.g. `v0.10.0`), then you can use the
+following command:
+
+```shell
+svn export https://github.com/vmware-tanzu/antrea/tags/<TAG>/build/yamls/elk-flow-collector/
```

Comment:
I don't mind too much the use of subversion here, but some folks may not be used to it or may not have it installed (I don't have it on my Linux VM for example).

A few possible alternatives:
 * `git clone --depth 1 --branch <TAG>|master https://github.com/vmware-tanzu/antrea.git`
 * write a helper script which takes care of downloading the files and creating the k8s resources: `curl https://raw.githubusercontent.com/vmware-tanzu/antrea/<TAG>/hack/deploy-elk.sh -O deploy-elk.sh && chmod +x deploy-elk.sh && ./deplo-elk.sh <TAG>`. But that's probably not worth the maintenance effort and I don't know how I feel about yet another script.

Reply:
Added clone approach as an alternative to the subversion utility.  Yes, another script may give a feeling of complicated steps.